recipes/emsdk/all/test_package/conanfile.py

```diff
@@ -0,0 +1,24 @@
+from conans import ConanFile, CMake, tools, RunEnvironment
+import os
+
+required_conan_version = ">=1.36.0"
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "build_type", "arch", "compiler"
+    generators = "cmake"
+    test_type = "build_requires"
```

Comment:
This is the reason why the M1 build is failing.
This is essentially requiring c3i to build emsdk twice: once for the build machine (=x86_64) and once for the host (=armv8).

I think, with the current behavior of conan/c3i of `test_type = "build_requires"`, it should not be used for the moment.

The only case where it is possible to be used if a package is os/arch/compiler-agnostic.